src/NodaTime/Calendars/HebrewScripturalCalculator.cs

```diff
@@ -171,17 +171,11 @@ internal static int GetDaysFromStartOfYearToStartOfMonth(int year, int month)
 
         internal static int DaysInMonth(int year, int month) => month switch
         {
-            // FIXME: How do we express multiple cases in a switch expression?
-            // We want: (2, 4, 6, 10, 13) => 29,
-            2 => 29,
-            4 => 29,
-            6 => 29,
-            10 => 29,
-            13 => 29,
+            var m when m == 2 || m == 4 || m == 6 || m == 10 || m == 13 => 29,
             8 => IsHeshvanLong(year) ? 30 : 29,
             9 => IsKislevShort(year) ? 29 : 30,
             12 => IsLeapYear(year) ? 30 : 29,
-            _ => 30 // 1, 3, 5, 7, 11, 13
+            _ => 30 // 1, 3, 5, 7, 11
```

Comment:
This was possibly a comment typo ?
the `13` case were already handled directly.

Reply:
Yes, this was a typo. I'm fine for there to be a typo fix.